오류 발생 시간: 2025-04-05 12:27:08.661598
오류 메시지: `output_signature` must contain objects that are subclass of `tf.TypeSpec` but found <class 'list'> which is not.

Traceback (most recent call last):
  File "C:\Users\iksun\.Programming\Project\personalColor\dual_input_model.py", line 596, in main
    history = model.fit(
              ^^^^^^^^^^
  File "C:\Users\iksun\AppData\Roaming\Python\Python312\site-packages\keras\src\utils\traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\iksun\AppData\Roaming\Python\Python312\site-packages\tensorflow\python\data\ops\from_generator_op.py", line 124, in _from_generator
    raise TypeError(f"`output_signature` must contain objects that are "
TypeError: `output_signature` must contain objects that are subclass of `tf.TypeSpec` but found <class 'list'> which is not.
